### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by seromarin**
## Summary
- Navbar app completed with fixed position.

## Screenshot
Captura de Pantalla 2020-10-30 a la(s) 5 18 23 p  m

## Trello Card
https://trello.com/c/R1rbzoGu/31-navbar

### CODE REVIEW

File: maquetacion/src/scss/_navbar.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  background-color: #FFF;
+  border-top: 6px solid #00ADEE;
+  box-shadow: 0 4px 10px 0 rgba(0,0,0,0.5);
+  position: fixed;
+  width: 100%;
+  top: 0;
+  z-index: 1;
+  
+  &__content {
+    width: 100%;
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_navbar.scss]: order by name

File: maquetacion/src/scss/_navbar.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 70px;
+  }
+
+  &__logo {
+    height: 70px;
+    width: 212.42px;
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_navbar.scss]: Why decimals?

File: maquetacion/src/scss/_navbar.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  }
+
+  &__logout {
+    // height: 18px;
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_navbar.scss]: Remove al comments

File: maquetacion/src/scss/_styles.scss [hunk 4]
```diff
@@ -6,18 +6,20 @@
...
+html, body {
+  height:100%;
 }
 
-.text-light {
-  color: $grey-light;
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_styles.scss]: Order by name

**Review by idcuadrowolox — CHANGES_REQUESTED** [review]

File: maquetacion/src/scss/_navbar.scss (hunk 1 above)
Comment (seromarin) [reply maquetacion/src/scss/_navbar.scss]: Done

File: maquetacion/src/scss/_navbar.scss (hunk 2 above)
Comment (seromarin) [reply maquetacion/src/scss/_navbar.scss]: Height remove and width changed to 215px

File: maquetacion/src/scss/_navbar.scss (hunk 3 above)
Comment (seromarin) [reply maquetacion/src/scss/_navbar.scss]: All comments removed in _navbar.scss

File: maquetacion/src/scss/_styles.scss (hunk 4 above)
Comment (seromarin) [reply maquetacion/src/scss/_styles.scss]: Done

**Review by idcuadrowolox — APPROVED** [review]

**Pull request merged**